python/core/unified_api_server.py

```
@@ -2088,22 +2358,35 @@ def _canonical_user_id(user_id: str, platform: str) -> str:
     Store and query usernames in lowercase for both platforms so we hit the same rows
     regardless of how the caller cased the name.
     """
+    if not user_id or not platform:
+        raise ValueError("user_id and platform cannot be empty")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)